packages/lib/RotatingLogs.test.ts

```diff
@@ -35,11 +35,28 @@ describe('RotatingLogs', () => {
 			await rotatingLogs.cleanActiveLogFile();
 			await msleep(1);
 			await rotatingLogs.deleteNonActiveLogFiles();
-			const files = await readdir(dir);
+			const files: string[] = await readdir(dir);
 			expect(files.find(file => file.match(/^log-[0-9]+.txt$/gi))).toBeFalsy();
 			expect(files.length).toBe(0);
 		} finally {
 			await remove(dir);
 		}
 	});
+
+	test('should not delete the log-timestamp.txt right after its be created', async () => {
+		let dir: string;
+		try {
+			dir = await createTempDir();
+			await createTestLogFile(dir);
+			await msleep(100);
```

Comment:
I defined a longer time to avoid its failure on more performactic machines. Because, to this test pass the time between `rotatingLogs.cleanActiveLogFile() and rotatingLogs.deleteNonActiveLogFiles();` needs to be less than or equal the time defined in RotatingLogs constructor. I believe that 100ms give us a big safety margin and its very quick anyway.